test/SlowTests/Server/Documents/AI/GenAi/GenAiBasics.cs

```diff
@@ -0,0 +1,533 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using Newtonsoft.Json;
+using Raven.Client;
+using Raven.Client.Documents.Operations.AI;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.OngoingTasks;
+using Raven.Server.Documents;
+using Raven.Server.Documents.ETL.Providers.AI.GenAi;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Server.Documents.AI.GenAi;
+
+public class GenAiBasics(ITestOutputHelper output) : RavenTestBase(output)
+{
+    [RavenFact(RavenTestCategory.Ai)]
+    public void CanCreateGenAiTask()
+    {
+        using var store = GetDocumentStore();
+        store.Maintenance.Send(new PutConnectionStringOperation<AiConnectionString>(new AiConnectionString
+        {
+            Name = "ollama-local",
+            Identifier = "ollama-local",
+            OllamaSettings = new OllamaSettings
+            {
+                Uri = "http://127.0.0.1:11434/",
```

Comment:
We have `RAVEN_AI_INTEGRATION_OLLAMA_URI` defined on Jenkins. See `OllamaConnectorForTesting`